meson.build

```diff
@@ -0,0 +1,31 @@
+project('template-dbus-service', 'cpp',
+    version : '0.1.0',
+    meson_version : '>=0.45',
+    default_options : [
+        'cpp_std=c++17',
```

Comment:
Is the template using C++17 features?

Reply:
Yes.

Reply:
E.g. std::optional, nested namespace definitions, constexpr variables are implicitly inlined... and... perhaps I missed something. Nothing big, but nice to have. Was not that much new in C++17 compared to C++14.

Reply:
To clarify a bit more if you worries are about support. It is all features GCC and Clang have supported for a very long time (at least ~2 years). You can check https://en.cppreference.com/w/cpp/compiler_support for an overview but have to look at release notes for GCC and Clang for more detailed information.

Reply:
Searching on https://layers.openembedded.org/layerindex/branch/master/layers/ and looking in the Poky repo with `gitk --all -- meta/recipes-devtools/gcc/` it looks to me like GCC 7.X (first major release May 2, 2017) is available for `thud`, `sumo` and even `pyro` but not `morty`.

Reply:
Ok, sounds good. Then we know we can build it with PELUX 1.0 since it's based on pyro, even though morty also should be supported in 1.0.
Let's use C++17 in this template.